tests/integration-tests/conftest.py

```diff
@@ -27,6 +27,7 @@
 
 import boto3
 import pytest
+import utils
```

Comment:
What about the Code Scanning comment below?

Reply:
Done